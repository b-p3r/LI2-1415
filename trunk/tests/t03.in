3 4
AAOE
ECRS
SVIA
