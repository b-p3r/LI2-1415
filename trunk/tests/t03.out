A
ACARO
ACAROS
ACO
ACOR
ACORES
ACOS
ACRE
AI
AIRES
AIS
AO
AOS
AR
ARA
ARAS
ARCA
ARCO
ARCOS
ARE
ARES
ARIA
ARIAS
ARO
AROS
AS
ASO
CA
CAES
CAO
CAOS
CARA
CARAS
CARO
CAROS
CE
CEAR
COA
COAR
COESA
COR
CORA
CORAIS
CORE
CORES
CORSA
COSA
COSE
COSER
COSI
COSIA
CRE
CRI
CRIA
CRIAS
CRISE
E
ECO
ECOA
ECOAR
ECOARA
ECOE
ECOS
ECRA
ECRAS
ERA
ERAS
ES
ESCOA
ESCOAR
ESCOE
ESCORA
ESCORAS
ESCORIA
ESCORIAS
IA
IAS
ICA
ICAR
ICARA
ICE
IR
IRA
IRAO
IRAS
IRE
IRES
IS
O
OCA
OCRE
OE
ORA
ORAI
ORAIS
ORAS
ORCA
ORCE
ORE
OS
RA
RACA
RACOES
RAS
RASO
RE
RES
RI
RIA
RIAS
RICA
RICO
RICOS
RIS
RISO
RO
ROA
ROCA
ROCE
ROSA
ROSE
SA
SAI
SAIR
SAIRA
SAIRAO
SARA
SARCA
SARE
SE
SEARA
SEARAS
SECA
SECAR
SECARA
SECO
SECOS
SER
SERA
SERAO
SERIA
SERICO
SERVE
SERVI
SERVIA
SERVICO
SI
SIR
SO
SOA
SOAR
SOARA
SOARIA
SOCA
SOCAR
SOCIA
SOE
SOR
SORVE
SORVIA
VE
VES
VIA
VIAS
VICE
VICOSA
VIR
VIRA
VIRAO
VIRAS
VIRE
VIRES
VIRO
VIROSE
VIS
VISA
VISAR
VISARA
VISARAO
VISE
VISO
VISOR
